internal/controllers/user_devices_controller.go

```diff
@@ -1919,6 +1994,30 @@ type AdminRegisterUserDevice struct {
 	Verified    bool    `json:"verified"`
 }
 
+type CompleteOAuthExchangeRequest struct {
```

Comment:
Can we add documentation strings and examples (see elsewhere) for these fields? Those will show up in the Swagger docs. I think that's a great help.

Reply:
Should we move this to the file that uses it?